lib/cdo/script_constants.rb

```diff
@@ -0,0 +1,54 @@
+# Info on existing tutorial scripts on Code Studio,
+# sometimes referenced in Pegasus.
+# Used for conditional behaviors.
+
+module ScriptConstants
+  HOC_2013_NAME = 'Hour of Code' # this is the old (2013) hour of code
```

Comment:
Ruby doesn't have enums so this is kind of how you do that. (You can define these types of constants in an array if you want them to be more enum-y) (just FYI, not necessarily a suggestion for this PR)

> On Dec 1, 2015, at 9:52, Mehal Shah <notifications@github.com> wrote:
> 
> In lib/cdo/script_constants.rb:
> 
> > @@ -0,0 +1,54 @@
> > +# Info on existing tutorial scripts on Code Studio,
> > +# sometimes referenced in Pegasus.
> > +# Used for conditional behaviors.
> > +
> > +module ScriptConstants
> > +  HOC_2013_NAME = 'Hour of Code' # this is the old (2013) hour of code
> I feel like there's a way to make this an enum, so we don't have to extend tests every time we write something new
> 
> —
> Reply to this email directly or view it on GitHub.
>